### PR TITLE
minor README typo

### PR DESCRIPTION
**Pull request by PixnBits**
not sure if the correct word is "as" or "to", made a guess, feel free to decline

### CODE REVIEW

**Comment by gholtslander-va** [comment]
Should probably be `to`, in my opinion. The code gets added to the source.

**Comment by everdimension** [comment]
@SpaceK33z 
The README is really a small puzzle currently. The opened issue is about this, too https://github.com/webpack-contrib/exports-loader/issues/9

I believe it should be *adds code **to** the file's source*. Please update.

Sorry to disturb if you're not currently maintaining this repo.

**Comment by graysongs** [comment]
_adds below code in the file's source._

**Comment by michael-ciniawsky** [comment]
@PixnBits Can you please close and reopen the PR to trigger the CLA bot again, it should be working now 😛

File: README.md [hunk 1]
```diff
@@ -10,12 +10,12 @@ npm install exports-loader
 
 ``` javascript
 require("exports-loader?file,parse=helpers.parse!./file.js");
-// adds below code the the file's source:
+// adds below code as the file's source:
```
Comment (michael-ciniawsky) [inline README.md]: `as` => `to` :)

**Pull request closed without merging**